backend/dal/Migrations/Initial/PostDeploy/01-NotificationTemplates.sql

```diff
@@ -996,6 +996,25 @@ INSERT INTO dbo.[NotificationTemplates] (
 <html><head><title>@Model.Environment.Title</title></head>
 <body><p>Dear Administrator,</p><p>@Model.AccessRequest.User.FirstName @Model.AccessRequest.User.LastName has submitted an access request to <a href="@Model.Environment.Uri">PIMS</a>.</p><p>Signin and review their request.</p></body></html>'
     , 'Access Request'
+), (
```

Comment:
This new template is why the DB needs to be refreshed.  If you don't want to refresh your DB you can just add this template manually.